File: week1_C/mario/mario.c
#include <cs50.h>
#include <stdio.h>

void printMario(int height);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while(height < 1);
    printMario(height);
}

void printMario(int height)
{

    for (int i = 0; i < height; ++i)
    {
        for (int j = 0; j < height; ++j)
        {
            if (j < height - i - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");

        for (int j = height; j > 0; --j)
        {
            if (j > height - i - 1)
            {
                printf("#");
            }
            else
            {
                break;
            }
        }

        printf("\n");
    }
}

File: week6_python/mario/mario.py
from cs50 import get_int
while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break


for i in range(h):
    for j in range(h):
        if j < h - i - 1:
            print(' ', end='')
        else:
            print('#', end='')

    print('  ', end='')

    for k in range(i + 1):
        print('#', end='')

    print()
